src/orderbook_fetcher.ts

```diff
@@ -17,12 +18,17 @@ export class OrderbookFetcher {
   intervalId: NodeJS.Timeout | null = null
 
   constructor(readonly web3: Web3, pageSize: number, pollFrequency: number, logger: CategoryLogger) {
+    orderBookFetcher.total.inc()
+    const endTimer = orderBookFetcher.durations.startTimer()
```

Comment:
I'm not very familiar with Prometheus metrics, but doesn't this timer need to start for each `poll` call, or else won't it will be measuring the time between when the fetcher was created and the latest poll?

Reply:
Great catch, you are right. I added the fetcher last minute before one 1<>1, and I rushed a bit the PR!  I'll address it right away